os/arch/arm/src/armv7-m/mpu.h

```diff
@@ -238,6 +238,34 @@ static inline void mpu_showtype(void)
 #endif
 }
 
+/****************************************************************************
+ * Name: mpu_get_access_type
```

Comment:
Please check the description also.
You changed the function name as `mpu_get_access_type_str` :)

Reply:
Done. Thanks.